aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -335,6 +335,52 @@ class QConv2dInt8 final : public c10::OperatorKernel {
     // Force output format to be NHWC
     // TODO: consider preserving input format
     // TODO: add MemoryFormat::ChannelsLast here once perf is fixed
+    auto input_scale = input_contig.q_scale();
+
+    // Re-quantizing the bias based on input scale and weight scale.
+    if (!pack_ptr.input_scale.has_value()) {
+      // Get the original weight and adjust it to uint8 from int8
+      auto weight_contig = pack_ptr.orig_weight;
```

Comment:
it's actually not contiguous - it's in channels_last :) I'd add `.contiguous(ChannelsLast)` here just to be sure (it's a noop if it's already in this format)